geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/IngestCommand.scala

```diff
@@ -27,19 +27,32 @@ class IngestCommand(parent: JCommander) extends Command(parent) with LazyLogging
   override val params = new IngestParameters()
 
   override def execute(): Unit = {
-    val fmt = Formats.fromString(Option(params.format).getOrElse(getFileExtension(params.files(0))))
+    val extensions = params.files.map(getFileExtension)
+    require(extensions.tail.forall(_ == extensions.head), "Input files must all be of the same file type")
```

Comment:
Would it be valid to say that if they don't specify a format, then all the files have to be the same type? Otherwise we base the ingest off the extension of the first file, which seems like it could lead to some issues and/or confusion.